lib/cli.rs

```diff
@@ -378,9 +381,18 @@ pub async fn get_balance(
 
 /// Retrieves an [`Account`] at a given [`Block`].
 ///
-/// `public_key` is the public key as a formatted string associated with the `Account`.
-///
 /// For details of other parameters, see [the module docs](crate::cli#common-parameters).
+///
+/// # Parameters
+/// - `maybe_rpc_id`: The optional RPC ID as a string slice.
+/// - `node_address`: The address of the node as a string slice.
+/// - `verbosity_level`: The verbosity level as a 64-bit unsigned integer.
+/// - `maybe_block_id`: The optional block ID as a string slice.
+/// - `account_identifier`: The account identifier as a string slice.
+///
+/// # Returns
+/// - `Result<SuccessResponse<GetAccountResult>, CliError>`: The result containing either a successful response with the account details or a `CliError`.
```

Comment:
I don't think we need to name the actual type here
```suggestion
/// The result containing either a successful response with the account details or a `CliError`.
```

Reply:
Changed thanks